src/ir_MilesTag2.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Victor Mukayev (vitos1k)
+/// @file
+/// @brief Support for the MilesTag2 IR protocol for LaserTag gaming
+/// @see http://hosting.cmalton.me.uk/chrism/lasertag/MT2Proto.pdf
+
+// Supports:
+//   Brand: Milestag2,  Model: Various
+
+// TODO(vitos1k): This implementation would support only
+// short SHOT packets(14bits) and MSGs = 24bits. Support
+// for long MSGs > 24bits is TODO
+
+#include <algorithm>
+#include "IRrecv.h"
+#include "IRsend.h"
+#include "IRutils.h"
+
+// Constants
+// Constants are similar to Sony SIRC protocol, bassically they are very
+// similar, just nbits are varying
```

Comment:
If I recall correctly, Sony's protocol also uses a different modulation frequency (40kHz vs 38kHz):
https://github.com/crankyoldgit/IRremoteESP8266/blob/8cf70043b7159e1ba672227fe13343cc3615e35e/src/ir_Sony.cpp#L35
and a different duty cycle 33% vs 25%:
https://github.com/crankyoldgit/IRremoteESP8266/blob/8cf70043b7159e1ba672227fe13343cc3615e35e/src/ir_Sony.cpp#L78
and Sony requires 3 messages (2 repeats).
https://github.com/crankyoldgit/IRremoteESP8266/blob/8cf70043b7159e1ba672227fe13343cc3615e35e/src/IRremoteESP8266.h#L1068

i.e. They differ more than just bit lengths.